xbmc/rendering/dx/DeviceResources.cpp

```diff
@@ -1097,3 +1112,159 @@ void DX::DeviceResources::Trim() const
 }
 
 #endif
+
+bool DX::DeviceResources::IsDisplayHDREnabled(bool* HDRCapable)
```

Comment:
bool* HDRCapable is only ever passed in as nullptr, and then its only use in here is to be assigned to hdrcapable at the end. Why do we even need this parameter then? Is there something im missing regarding this?

Reply:
Is just prepared for future functionality. Is explained in this commit message: https://github.com/xbmc/xbmc/pull/17012/commits/9f8e0919fe684a42f3a77b9caddbd7a82a441f47

Reply:
I'd take it out for now and put in a to-do comment instead

Reply:
Done